src/cmps/Json-input.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+
+
+
+
+function JsonInput() {
+    return (
+        <div className="Json-input-container">
+            JSON input main container
+
+            <form action="" method="post">
+                JSON input form container
+
+
+                {/* <textarea name="Json-input-text" id="" cols="30" rows="10"></textarea> */}
```

Comment:
remove comment and spaces

Reply:
comment was left intentionally for future use